policy/modules/roles/unconfineduser.te

```diff
@@ -65,6 +65,8 @@ typealias unconfined_t alias unconfined_crontab_t;
 dontaudit unconfined_t self:dir write;
 dontaudit unconfined_t self:file setattr;
 
+allow unconfined_t self:lockdown integrity;
```

Comment:
The `integrity` permission should rather be added to `allow unconfined_domain_type self:lockdown confidentiality;` in `domain.te`.

Reply:
Done.